.github/workflows/dapr-automerge.yml

```diff
@@ -0,0 +1,26 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+jobs:
+  automerge:
+    if: github.repository_owner == 'dapr'
+    name: Automerge and update PRs.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Automerge and update
+        env:
+          MAINTAINERS: yaron2,youngbupark,artursouza
```

Comment:
change maintainers.

Reply:
I thought I did. Odd. Let me see.

Reply:
I changed in the wrong repo :) That is why.